templates/template_data.go

```diff
@@ -57,6 +64,30 @@ type ExtendedData struct {
 	ExternalURL string `json:"externalURL"`
 }
 
+// FromContent calls Parse on all provided template content and returns the resulting Template. Content equivalent to templates.FromGlobs.
+func FromContent(templateContents []string, options ...template.Option) (*Template, error) {
+	// Create new template with only file-based defaults. Done this way to simplify transition to FromContent.
+	t, err := FromGlobs(nil, options...)
```

Comment:
I think it's fine to remove `FromGlobs` in this PR, then we don't need to have a no-op `FromGlobs`.

Reply:
It's not a no-op as it parses the default file-based templates `default.tmpl` and `email.tmpl`, but I can remove it in the PR.

Reply:
575da9a00f98e0eb2ecafab1a82c8b5879bf9c0d